app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -104,10 +183,10 @@ class Dashboard extends React.Component {
         // this.getGradeEntryFormColumnBreakdown();
       } else if (this.state.assessment_type === 'Assignment') {
         // TODO
-        $.ajax({
-          url: Routes.grade_distribution_graph_data_assignment_path(this.state.assessment_id),
-          dataType: 'json',
-        }).then(res => this.setState({data: res}))
+        // $.ajax({
```

Comment:
You've done a good job in moving the code, so in fact you can *delete* this entire `else if` branch now!